src/remixAppManager.js

```diff
@@ -202,6 +202,22 @@ export class RemixAppManager extends PluginEngine {
       icon: 'https://zokrates.blockchain-it.hr/zokrates.svg',
       location: 'sidePanel'
     }
+    const quorum = {
+      name: 'quorum',
+      displayName: 'Quorum Support',
```

Comment:
Feels like "Hot line support" ^^.
Also, some developers don't know what Quorum is. Maybe `Quorum Network` would be more explicit